src/WebJobs.Extensions.CosmosDB/Trigger/CosmosDBMetricsProvider.cs

```diff
@@ -61,6 +61,14 @@ public async Task<CosmosDBTriggerMetrics> GetMetricsAsync()
                 partitionCount = partitionWorkList.Count;
                 remainingWork = partitionWorkList.Sum(item => item.EstimatedLag);
             }
+            catch (CosmosException cosmosException) when (cosmosException.StatusCode == HttpStatusCode.Gone)
+            {
+                // Temporary handling of split issue described in https://github.com/Azure/azure-cosmos-dotnet-v3/issues/4285
+                // This happens if the main instance is not running, potentially using Consumption Plan
+                // In this case, we return a positive value to make the Scale Controller consider spinning at least a single instance
+                partitionCount = 1;
```

Comment:
Clarification: Idea is that it can only happen if the processor is not up and setting to some min-threshold will spin it-up right? 

For a large collection can the processor might take sometime to get caught-up to the impacted partition?

Reply:
Yes, setting it to at least 1 will force Scale Controller to spin 1 instance. That instance will hit the split exception and create the leases, then Scale Controller can decide to scale further up.

Reply:
> For a large collection can the processor might take sometime to get caught-up to the impacted partition?

It might but that's not important. Once the split is handled, then the Scale Controller Metrics will work again and produce valid metrics that can help scale up to more instances if needed